examples/docker-agent/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.6
```

Comment:
Was this file a mv from games/a/docker/Dockerfile?

Reply:
appears to be. can you change this so that it is a mv in order to keep the history more clean?

Reply:
Ping. Please do this for all the files that were moved.

Reply:
Ah, yes! I was trying to figure out how to do this. Initially when I moved the files, they were all showing as moved in the history. Perhaps when I modified them, they don't show it anymore. Any ideas how to do it?

Reply:
i think that the easiest way to do is to back these up (e.g. `mv examples/docker-agent/Dockerfile examples/docker-agent/Dockerfile.backup`) then `git mv /path/to/original /new/path/original`, then mv backup into new path.

Reply:
This method actually just puts the repo in the same state and hence no change. 
Seeing the commit history, until a certain point it showed that this file was a move. See [here](https://github.com/MultiAgentLearning/playground/pull/26/commits/73ac676b8ae0dc5efe07d2fc964752a28deb98e7) when it stopped showing. Before this commit, it shows it was a move.